template.go

```diff
@@ -129,4 +129,9 @@ func (r {{$typename}}) Value() (driver.Value, error) {
 
 {{end}}
```

Comment:
what's this {{end}}? Does your MarshalCSV func need to be before that?

Reply:
😬  good catch 472af33